mantidproject/mantid
# ISIS SANS improve error messages

**Description of work.**
This updates the sans gui so that when an error is raised during file finding or reduction a box pops up to warn the user rather than just writing out to the log which is often not noticed. 

**Report to:**[nobody]. 

**To test:**
Get this data
[loqdemo.zip](https://github.com/mantidproject/mantid/files/2118722/loqdemo.zip)
Open the new sans GUI, select SANS2D as the instrument
Load MaskFile.txt as the user file 
Load batch_mode_reduction.csv as the batch file
Confirm that an error box appears.


Fixes #21115. 

Requires release notes which are here #22628

Release notes are updated in #PR to reduce merge conflicts.
---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
